resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -151,12 +151,25 @@ class="fas fa-info-circle"></i></a></b>
                 </div>
 
                 <div class="row">
-                    <div class="col-3">
+                    <div class="col-4">
                         <button class="btn btn-success" type="submit">Save Profile</button>
                     </div>
+
+                    @toggle('discord_role_updater')
+                        @if(Auth::user()->discord)
+                            <div class="col-4">
+                                <a href="/dashboard/controllers/profile/discord" class="btn btn-success" type="button">Update Discord Role</a>
+                            </div>profile.
+                        @else
+                            <div class="col-4">
+                                <a href="#" data-toggle="tooltip" title="No Discord Role Found: Ensure your Discord ID is linked to your VATUSA account. The roster may take time to refresh. You can manually update your role in the ZTL Discord server in the meantime." class="btn btn-secondary" type="button">Update Discord Role</a>
```

Comment:
Again, please make the button text "Update Discord Roles"
Recommend that you update the tooltip text to be slightly more descriptive... instead of "No Discord Role Found" make it "No Discord ID Found"